cherrypy/lib/httputil.py

```diff
@@ -66,6 +68,31 @@ def protocol_from_http(protocol_str):
     return int(protocol_str[5]), int(protocol_str[7])
 
 
+def passes_if_range_check(request):
```

Comment:
This is inconsistent with `get_ranges`, which accepts already retrieved header data.

Reply:
I will definitely remove that. Sorry, I left in some development stuff!

Reply:
It's fine, you don't have to apologize! Nobody's judging you :)